src/domain/context/aspect/dto/aspect.dto.update.ts

```diff
@@ -10,6 +10,10 @@ export class UpdateAspectInput extends UpdateNameableInput {
   @MaxLength(LONG_TEXT_LENGTH)
   description?: string;
 
+  @Field({ nullable: true })
```

Comment:
No need to update the type description; also impacts the updateAspect method

Reply:
So if somebody adds a type description and e.g. copy / pastes some description incorrectly, the option to correct it won't exist? Is that really what we want?